pkg/apis/certmanager/v1/types.go

```diff
@@ -143,6 +143,18 @@ const (
 	VenafiPickupIDAnnotationKey = "venafi.cert-manager.io/pickup-id"
 )
 
+// TODO
+const (
+	// TODO
+	CertificateSigningRequestDurationAnnotationKey = "cert-manager.io/request-duration"
+
+	// TODO
+	CertificateSigningRequestIsCAAnnotationKey = "cert-manager.io/request-is-ca"
+
+	// TODO
+	CertificateSigningRequestCAAnnotationKey = "cert-manager.io/ca"
```

Comment:
Can we move these into some kind of experimental API group? I don't think we want to call them `v1` just yet :)

Reply:
Yep, good idea.